include/openenclave/bits/edl/syscall_types.h

```diff
@@ -4,11 +4,39 @@
 #ifndef _OE_EDL_SYSCALL_TYPES_H
 #define _OE_EDL_SYSCALL_TYPES_H
 
+#include <openenclave/bits/defs.h>
+
 /* DISCLAIMER:
  * This header is published with no guarantees of stability and is not part
  * of the Open Enclave public API surface. It is only intended to be used
  * internally by the OE runtime. */
 
 typedef int64_t oe_host_fd_t;
 
+typedef union _oe_epoll_data_t {
```

Comment:
Just found that we use this format only for union (although I personally like this better).

Reply:
I think some of these syscall types are intended to match the actual syscall struct name. For example `oe_epoll_event` matches the linux structure `epoll_event`.

But I do agree the _t format is preferrable. POSIX has some weird things because they don't use a type suffix. For example `stat` is the name of both a function and a struct.

Reply:
I'm actually refer to
```
union _oe_epoll_data_t {
...
};
```
instead of
```
union _oe_epoll_data_t 
{
...
};
```

Reply:
oh, I didn't even notice that. We should probably fix that in clang-format :smile: